security-center/snippets/package.json

```diff
@@ -9,7 +9,7 @@
     "node": ">=16.0.0"
   },
   "scripts": {
-    "test": "c8 mocha -p -j 2 --recursive --timeout 6000000 system-test/v2/findings.test.js"
+    "test": "c8 mocha -p -j 2 --recursive --timeout 6000000 system-test/v2/"
```

Comment:
issue: The test command for security center snippets overall should run all snippet tests, not just v2.

Reply:
Addressed.